primitives/primproc/primitiveserver.cpp

```diff
@@ -1147,6 +1147,7 @@ int DictScanJob::operator()()
     {
       loadBlock(cmd->LBID, verInfo, cmd->Hdr.TransactionID, cmd->CompType, data, &wasBlockInCache,
                 &blocksRead, fLBIDTraceOn, session);
+      // TODO MCOL-5021
```

Comment:
DictScan always has a ColumnCommand in front for its token column of it so you don't need to add support here.

Reply:
This makes sense. I have removed this TODO.